src/components/application_manager/test/commands/mobile/simple_notification_commands_test.cc

```diff
@@ -68,6 +69,7 @@ typedef Types<commands::OnAppInterfaceUnregisteredNotification,
               commands::OnAudioPassThruNotification,
               commands::OnLanguageChangeNotification,
               commands::OnPermissionsChangeNotification,
+              commands::mobile::OnSeekMediaClockTimerNotification,
```

Comment:
@indlu Well if I am correct, squashing is necessary for commits that are supplementing some already existing commit. As for the unit tests they are related to separate task, and therefore should have their own separate commit.